lib/src/main/java/com/cloudbees/groovy/cps/impl/NotBlock.java

```diff
@@ -34,6 +34,7 @@ public Next receive(Object o) {
         return new ContinuationImpl(e, k).then(b, e, cast);
     }
 
+    @SuppressFBWarnings(value = "SIC_INNER_SHOULD_BE_STATIC", justification = "Common Jenkins pattern that inner classses are not static.")
```

Comment:
No,

> This class is an inner class, but does not use its embedded reference to the object which created it.  This reference makes the instances of the class larger, and may keep the reference to the creator object alive longer than necessary.  If possible, the class should be made static.

so that is really a bug…but not one that we can fix compatibly now (since builds may be running across the upgrade). So
```suggestion
    @SuppressFBWarnings(value = "SIC_INNER_SHOULD_BE_STATIC", justification = "Too late to fix compatibly.")
```